### PR TITLE
Enable multi-process training on CPU

### PR DESCRIPTION
**Pull request by dirkgr**
This is probably only useful during testing.

### CODE REVIEW

**Comment by dirkgr** [comment]
@epwalsh, can you review this? Especially the way the tests are configured?

**Comment by epwalsh** [comment]
Yeup! I’ll take a look right after lunch

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Additional CI checks to ensure docstrings are consistently formatted.
+- Ability to train on CPU with multiple processes
```
Comment (epwalsh) [inline CHANGELOG.md:20]:
```suggestion
- Ability to train on CPU with multiple processes by setting `cuda_devices` to a list of negative integers in your training config. For example: `"distributed": {"cuda_devices": [-1, -1]}`. This is mainly to make it easier to test and debug distributed training code..
```

**Comment by dirkgr** [comment]
> LGTM! Looks like there are some multi-gpu tests in tests/training/trainer_test.py that might make sense to mark as `@cpu_or_gpu` as well?

There is only one, and it's about raising an error, so I want to leave that one alone.

**Pull request merged**